### ISSUE TITLE
"Half Class Table" fails to render correctly when printing in Chrome

### ISSUE THREAD

**Issue by calculuschild** [stolksdorf/homebrewery#130]
**If you are suggestting a feature or improvement**, delete everything here and just write your idea.

If you are reporting a bug, please fill out the details. If you don't I will most likely just close the issue asking for more detail.

---

**Browser Type/Version**: Google Chrome Version 51.0.2704.84 m

**Operating System**: Windows 10 64x

**Issue Description**: "Half Class Table" and "Class Table" render ok in the preview pane, but when printing to PDF only the borders are styled, and all of the table content is rendered as plain text instead of formatted as a table.

**Brew code to reproduce**:

```
<div class='classTable'>
##### The Manicurist
| Level | Proficiency Bonus | Features | Malefic Chemist|
|:---:|:---:|:---|:---:|
| 1st | +2 | ─ | +1 |
| 2nd | +2 | Gunslinger Corruptor | +1 |
| 3rd | +2 | ─ | +2 |
| 4th | +2 | Ability Score Improvement, Gunslinger Corruptor | +2 |
| 5th | +3 | ─ | +2 |
| 6th | +3 | Ability Score Improvement, Police Necromancer | +3 |
| 7th | +3 | Biochemical Sorcery | +3 |
| 8th | +3 | Ability Score Improvement | +4 |
| 9th | +4 | Statistical Occultism | +4 |
| 10th | +4 | Genetic Banishing | +4 |
| 11th | +4 | Astrological Botany | +4 |
| 12th | +4 | Ability Score Improvement | +4 |
| 13th | +5 | Infernal Banker | +4 |
| 14th | +5 | Ability Score Improvement, Civil Alchemy | +4 |
| 15th | +5 | Astrological Chemistry | +4 |
| 16th | +5 | Ability Score Improvement | +4 |
| 17th | +6 | Spiritual Biochemistry | +5 |
| 18th | +6 | Civil Alchemy | +6 |
| 19th | +6 | Ability Score Improvement, Malefic Chemist | +7 |
| 20th | +6 | Genetic Banishing | +8 |
</div>
```

**Related Images** : 
![image](https://cloud.githubusercontent.com/assets/5878534/15844666/64492b30-2c29-11e6-872d-3e45263dce07.png)
(Side-pane preview)

![image](https://cloud.githubusercontent.com/assets/5878534/15844686/7d1026aa-2c29-11e6-901a-71e599281bbb.png)
(Print preview)

**Comment by stolksdorf** [comment]
Ah, I know _exactly_ what's happening. Thanks for the bug report! I'll see if I can quickly fix it before work.

**Comment by stolksdorf** [comment]
Okay pushed out a fix in v2.2.2, let me know if it does the trick!

**Comment by calculuschild** [comment]
Seems to render ok now!  Unfortunately I am now unable to save my changes to any projects.  See the new issue I created.
